WPiOS Integration.md

```diff
@@ -0,0 +1,53 @@
+# WordPress iOS Integration Guide
+
+## How to work with local gutenberg-mobile checkout
+
+The `LOCAL_GUTENBERG` environment variable is useful when you want to iterate on the native iOS code (Swift, Objective-C) in `gutenberg-mobile` while testing it inside [WPiOS](https://github.com/wordpress-mobile/WordPress-iOS) app. This also includes the case when you update a dependency in `gutenberg-mobile` that has native code in it. This flag plays no role in which WPiOS run scheme will be selected or whether Metro will be used or not, but it will affect which JS bundle WPiOS will use. In case the flag is enabled, it will be the one in locally available checkout of the `gutenberg-mobile` repo. That's why, when using this flag, it would be better to run the Metro server or recreate JS bundles in local `gutenberg-mobile` checkout so that the native and JS code are in sync. 
```

Comment:
```suggestion
The `LOCAL_GUTENBERG` environment variable is useful when you want to iterate on the native iOS code (Swift, Objective-C) in `gutenberg-mobile` while testing it inside the [WPiOS](https://github.com/wordpress-mobile/WordPress-iOS) app. This also includes the case when you update a dependency in `gutenberg-mobile` that includes native code.

The `LOCAL_GUTENBERG` flag sets which JS bundle WPiOS uses. It does not impact which WPiOS run scheme is selected or whether or not Metro is used. When the flag is enabled, the bundle available within the local `gutenberg-mobile` checkout is used. That's why, when using this flag, it is better to run the Metro server or recreate JS bundles in the local `gutenberg-mobile` checkout so that the native and JS code are in sync. 
```

Reply:
Much better, thanks!